tests/utils.go

```diff
@@ -3863,14 +3863,18 @@ func GenerateHelloWorldServer(vmi *v1.VirtualMachineInstance, testPort int, prot
 	Expect(err).ToNot(HaveOccurred())
 }
 
-func UpdateClusterConfigValue(key string, value string) {
+// UpdateClusterConfigValueAndWait updates the given configuration in the kubevirt config map and then waits
+// for the given duration, to allow the configuration events to be propagated to the consumers.
+func UpdateClusterConfigValueAndWait(key string, value string, howLong time.Duration) {
 	virtClient, err := kubecli.GetKubevirtClient()
 	PanicOnError(err)
 	cfgMap, err := virtClient.CoreV1().ConfigMaps(KubeVirtInstallNamespace).Get(kubevirtConfig, metav1.GetOptions{})
-	Expect(err).NotTo(HaveOccurred())
+	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 	cfgMap.Data[key] = value
 	_, err = virtClient.CoreV1().ConfigMaps(KubeVirtInstallNamespace).Update(cfgMap)
-	Expect(err).ToNot(HaveOccurred())
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
+
+	time.Sleep(howLong)
```

Comment:
I see you are using 2 and 5 seconds as timeout, why are they different? Wouldn't it be easier to use the same for all and just hardcode it here?

Reply:
Yes, it was one of the things I was uncertain about.

The current value (when applied) was 5 seconds, which seems like a lot to me so I did not want to put a lot of 5 seconds here and there. On the other hand, I was not sure about lowering the current sleep time where applied.

It's also true that we can put a constant and tune until we find a one size / fits all. Shall I try with 2 seconds?

Reply:
> Shall I try with 2 seconds?

yes, let's try that and have a careful look at test results :+1: